base_local_planner/src/point_and_shoot_trajectory_generator.cpp

```diff
@@ -227,8 +227,16 @@ Eigen::Vector3f PointAndShootTrajectoryGenerator::computeNewVelocities(const Eig
   double bearing = angles::normalize_angle(std::atan2(dy, dx) - pos[2]);
 
   // Simple proportional control with limits
-  double desired_angular_velocity = bearing / std::fabs(bearing)
+  double desired_angular_velocity;
```

Comment:
It's a good idea to always initialize variables so that you don't get weird bugs when refactoring later.  Could do this as:

double desired_angular_velocity = 0;
if (std::fabs(bearing) > 0.00001) {
  d_a_v = bearing / ..... (rest of equation)
}

Reply:
sure makes sense. 
Updating the PR now.